### ISSUE TITLE
Automatic configuration wrongly detects `.only` everywhere in the test files

### ISSUE THREAD

**Issue by MickVanDuijn** [wallabyjs/public#2509]
### Issue description or question

When using `.only` anywhere in file, Wallaby will only run that file. This clashes with the syntax of some libraries (in my case [Joi](https://github.com/sideway/joi)), but is not limited to actual code occurrences of `.only()` (it's also affected by comments).

To reproduce, create two `.spec.ts` files, run with automatic configuration (projects with a `wallaby.js` custom configuration file aren't affected), and place a `// .only()` comment in one of them like so:
```
describe('foo', () => {
    it('should run all other tests as well', () => {
        // .only()
        expect('foo').toEqual('foo')
    })
})
```

### Wallaby diagnostics report

Not applicable

**Comment by smcenlly** [comment]
Thanks for creating an issue. We've enhanced our `.only()` detection mechanism, which will address your problem.

This fix is available in the updated Wallaby core runtime, `v1.0.952`. Wallaby should update automatically within a few minute with the latest version. If you want Wallaby to update immediately, you may [refer to our docs](https://wallabyjs.com/docs/intro/troubleshooting.html#forcing-wallabyjs-core-update) for details on how to force a core update.

**Comment by MickVanDuijn** [comment]
Thanks a lot for the fast resolution! I _had_ written a workaround locally in my project, which I can happily remove again before even commiting it! 🚀
